student.rb

```diff
@@ -18,5 +18,5 @@ def add_classroom
   end
 end
 
-std = Student.new(16, 'History', 'Michal')
+std = Student.new(18, 'Math', 'Olivier')
 p std.can_use_services?
```

Comment:
- [ ] [OPTIONAL] Please remove any unnecessary code to make your code cleaner and more professional.